src/core/chart.js

```diff
@@ -0,0 +1,252 @@
+import _ from 'underscore';
+
+
+var PRECISION = 2;
+
+/* This function takes one or more JSON objects from the 
+ * "timeseries" API call with this format:
+ * 
+ * {
+ * "id": "tasmax_mClim_BCCAQv2_bcc-csm1-1-m_historical-rcp45_r1i1p1_20700101-20991231_Canada",
+ * "units": "degC",
+ * "data": {
+ *   "2085-01-15T00:00:00Z": -17.498223073165622,
+ *   "2085-02-15T00:00:00Z": -15.54878007851129,
+ *   "2085-03-15T00:00:00Z": -11.671093808333737,
+ *                    ...
+ *    }
+ * }
+ * 
+ * and return a C3 graph object displaying them.
+ * 
+ * It takes an arbitrary number of data objects, but no more than
+ * two separate unit types. Allowable data resolutions are monthly(12), 
```

Comment:
Does it allow both timeseries and climatologies? By the 12/4/1, I would assume only climatologies, but it would be good to clarify.